src/main/webapp/style.css

```
@@ -38,3 +38,7 @@ ul {
 
   line-height: 1.4em;
 }
+
+#editprofile{
```

Missing a space:

```suggestion
#editprofile {
```

Its helpful to always run your code with the autoformatters before committing the changes.